src/IterableValueStack.php

```diff
@@ -38,7 +38,7 @@ public function push(callable $modifier): self
     }
 
     /**
-     * @phpstan-param array<int, (callable(iterable<TKey, TValue> $value): iterable<TKey, TValue>)> $modifiers
+     * @phpstan-param array<int, (callable(iterable<TKey,TValue>): iterable<TKey,TValue>)> $modifiers
```

Comment:
The space between `:` and `iterable` should be omitted. If it was changed in other places, so should here also.